system-int-test/test/cmr/system_int_test/ingest/tool_ingest_test.clj

```diff
@@ -217,3 +218,50 @@
                              :native-id "tool_v_1.0"
                              :format "application/vnd.nasa.cmr.umm+json; version=1.0"})]
       (is (= 201 status)))))
+
+(deftest related-url-content-type-type-and-subtype-check
+  (testing
+    "Check valid and invalid related urls and url's url content type, type, and subtype.
```

Comment:
Can you turn this into a comment and make the "testing" form more concise.

Reply:
Sure. Thanks.

Reply:
Sure. Thanks.